net/nimble/controller/src/ble_ll_scan.c

```diff
@@ -344,6 +359,26 @@ ble_ll_scan_find_dup_adv(uint8_t *addr, uint8_t txadd)
 }
 
 /**
+ * Do scan machine clean up on PHY disabled
+ *
+ */
+void
+ble_ll_scan_clean_on_phy_disabled(void)
```

Comment:
I'd just call it ble_ll_scan_clean_aux_data or sth and use wherever appropriate - I saw this cleanup in few other places in the code iirc...